tests/nosetests/pyanaconda_tests/module_storage_test.py

```diff
@@ -1467,11 +1467,6 @@ def mount_filesystems_test(self, blivet_run_program, makedirs, core_run_program,
         blivet_run_program.return_value = (0, "")
         core_run_program.return_value = (0, "")
         MountFilesystemsTask(storage).run()
-        # created the mount points
-        makedirs.assert_any_call('/mnt/sysimage/dev', 0o755)
-        # sysimage mounts happened
-        blivet_run_program.assert_any_call(
-                ['mount', '-t', 'bind', '-o', 'bind,defaults', '/dev', '/mnt/sysimage/dev'])
```

Comment:
I guess these mockings still need to happen, even if you don't check them?

Reply:
Yes, we have to mock these calls to prevent the actual mounting.

Reply:
Yes, the calls must be mocked out, as actually doing them is both forbidden in containers and actively harmful on developer machines.

However, OOI: How is /dev target/architecture specific? I can't imagine a platform (aside from building a simple chroot maybe) where /dev isn't required. If /dev somehow gets in the way, then perhaps let's test /run? That is a firm OS requirement these days (/run *must* be a tmpfs).

FTR, I added these as the marching orders were "mock them out, but make sure the mounts will happen". So, not much objection to dropping this, but it improves coverage and I'm mostly curious where they get in the way. Thanks!

Reply:
This check was failing when I ran unit tests locally in my mock environment, but I am not able to reproduce it anymore. Weird. I have removed this commit from the pull request.